src/pages/components/menu.mdx

```diff
@@ -21,9 +21,16 @@ components:
 <Usage>
   <Use>
     <ul>
-      <li>To display a list of choices, such as options or actions</li>
+      <li>To let users see and choose one thing from a list of items</li>
     </ul>
   </Use>
+  <Misuse>
+    <ul>
+      <li>To display a list of actions use Split Button instead</li>
+      <li>To help users navigate use Anchors instead</li>
+      <li>To move between views use Tabs instead</li>
```

Comment:
Curious if we're consistent on whether we make these links or not? I'm looking at `Split Button`, `Anchors`, and `Tabs`. Surrounding the text with `<Markdown>` allows you to use MD link syntax.